### ISSUE TITLE
Importing an empty file broken in stylus package

### ISSUE THREAD

**Issue by ccorcos** [meteor/meteor#5314]
Just updated and I get this error:

```
   While processing files with stylus (for target web.browser):
   client/main.styl: Stylus compiler error: client/main.styl:67:9
   63| // My imports
   64| @import 'styles/utils.import'
   65| @import 'styles/animate.import'
   66| @import 'styles/input.import'
   67| @import 'playground.import'
   ---------------^
   68| @import 'views/svg/svg.import'
   69| @import 'views/components/layout.import'
   70| @import 'views/components/checkbox.import'

   ENOENT, no such file or directory '{}/client/playground.import.styl'
```

**Comment by ccorcos** [comment]
I commented it out and got another error.

```
   While processing files with stylus (for target web.browser):
   client/main.styl: Stylus compiler error: client/main.styl:80:9
   76| @import 'views/components/event-item.import'
   77| @import 'views/components/user-info.import'
   78| @import 'views/components/list.import'
   79| @import 'views/login.import'
   80| @import 'views/user.import'
   ---------------^
   81| @import 'views/users.import'
   82| 
   83| color-duration = 0.1s
   
   ENOENT, no such file or directory '{}/client/views/user.import.styl'
```

Both files were empty -- looks like you cant import an empty file...

**Comment by stubailo** [comment]
OK just tried this myself and confirmed!

It actually works properly with LESS but not Stylus, so it might be an error in that package specifically somehow.

**Comment by abernix** [comment]
Thanks for originally reporting this bug in Meteor!  I created [this repo](https://github.com/abernix/meteor-issue-5314) to show that this is fixed in Meteor 1.4 (and I confirmed it was broken on the same repo when in Meteor 1.3.2.4).  There have been similar upstream `stylus` fixes including https://github.com/stylus/stylus/commit/e740fceae3305ec4532a6c58071eb1b9240f1454 or https://github.com/stylus/stylus/commit/bba7b258e61b19d7527316b787e4967bc74b039f but I actually believe this was fixed by general 1.4 changes as neither of those would have been included in Meteor 1.4.0.1 which I also tested.

Closing as fixed! 🎉
